guides/request_lifecycle.md

```diff
@@ -273,7 +283,9 @@ And this is what the template should look like:
 </section>
 ```
 
-Our messenger appears as `@messenger`. We call "assigns" the values passed from the controller to views. It is a special bit of metaprogrammed syntax which stands in for `assigns.messenger`. The result is much nicer on the eyes and much easier to work with in a template.
+Our messenger appears as `@messenger`.
+
+The values we passed to the view from the controller are collectively called our "assigns". We could access our messenger value via `assigns.messenger` but through some metaprogramming, Phoenix gives us the much cleaner `@` syntax for use in templates.
```

Comment:
"We call "assigns" the values passed from the controller to views" read a bit backwards.